unified-installer/astra-unified-installer.sh

```diff
@@ -1857,11 +2077,11 @@ step_check_volumesnapshotclasses() {
 }
 
 step_check_namespace_and_pull_secret_exist() {
-    local -r namespace="$NAMESPACE"
-    local -r pull_secret="$IMAGE_PULL_SECRET"
+    local -r namespace="$1"
+    local -r pull_secret="${2:-""}"
```

Comment:
Made more generic so that it can be called for both the connector and trident namespace.